src/main.jsx

```diff
@@ -13,6 +13,8 @@ import ErrorPage from "./pages/Error/ErrorPage";
 import HomePage from "./pages/Home/HomePage";
 import LinkedInLearning from "./pages/LinkedInLearning/LinkedInLearningPage";
 import LoginPage from "./pages/Login/LoginPage";
+import CCMPage from "./pages/CompanyCourseManagement/CCMPage";
+import CTOPage from "./pages/CompanyTrainingOverview/CTOPage";
```

Comment:
Please don't use Abbreviations for the page names

Reply:
Page names have been updated to full names